packages/core/src/components/Autocomplete/Autocomplete.jsx

```diff
@@ -188,6 +191,14 @@ Autocomplete.propTypes = {
    * Clear search text that will appear on the page as part of the rendered `<button>` component
    */
   clearInputText: PropTypes.node,
+  /**
+   * Used to focus child `TextField` on `componentDidMount()`
+   */
+  focusTrigger: PropTypes.bool,
+  /**
+   * A unique `id` to be passed to the child `TextField`.
```

Comment:
Might be worth it to add something like "if none is provided, an id will be auto-generated" since this prop isn't required and it may be a little confusing